### PR TITLE
Check the floating error

### LINKED ISSUE #3888: Floating error to check remaining in the apex model

### PR DESCRIPTION
**Pull request by byzheng**
resolves  #3888

### CODE REVIEW

**Comment by hol430** [comment]
retest this please jenkins

**Comment by hol430** [comment]
retest this please jenkins

File: Models/Plant/Structure/ApexBase.cs [hunk 1]
```diff
@@ -239,11 +239,11 @@ protected void OnDoPotentialPlantGrowth(object sender, EventArgs e)
                         double remove = Math.Min(apexGroupSize[i], remainingRemoveApex);
                         apexGroupSize[i] -= remove;
                         remainingRemoveApex -= remove;
-                        if (remainingRemoveApex <= 0.0)
+                        if (remainingRemoveApex <= 0.00001)
```
Comment (hol353) [inline Models/Plant/Structure/ApexBase.cs]: Why not use MathUtilities.IsLessThan(remainingRemoveApex, 0)  ??

**Comment by hol353** [comment]
@byzheng Can you respond to my review query above?

File: Models/Plant/Structure/ApexBase.cs (hunk 1 above)
Comment (byzheng) [reply Models/Plant/Structure/ApexBase.cs]: @hol353 There is a floating error here. The remainingRemoveApex could be more than zero (e.g. 0.00001)

**Pull request merged**